rosdep/python.yaml

```diff
@@ -3674,6 +3674,17 @@ python-vlc-pip:
   ubuntu:
     pip:
       packages: [python-vlc]
+python-voluptuous:
+  debian:
+    jessie:
+      pip:
+        packages: [voluptuous]
+    stretch: [python-voluptuous]
+  fedora:
+    pip:
+      packages: [voluptuous]
```

Comment:
This package is also available in the fedora package database: https://admin.fedoraproject.org/pkgdb/package/rpms/python-voluptuous/